File: trunk/Dominio/src/br/cefetrn/datinf/estoque/dominio/Cartao.java
package br.cefetrn.datinf.estoque.dominio;

import java.io.Serializable;
import java.util.Date;

@SuppressWarnings("serial")
public class Cartao implements Serializable{
	private long id;
	private int digito;
	private Date vencimento;
	private Cliente cliente;
	private double limite;
	
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public int getDigito() {
		return digito;
	}
	public void setDigito(int digito) {
		this.digito = digito;
	}
	public Date getVencimento() {
		return vencimento;
	}
	public void setVencimento(Date vencimento) {
		this.vencimento = vencimento;
	}
	public Cliente getCliente() {
		return cliente;
	}
	public void setCliente(Cliente cliente) {
		this.cliente = cliente;
	}
	public double getLimite() {
		return limite;
	}
	public void setLimite(double limite) {
		this.limite = limite;
	}
}

File: trunk/eVendas-Site/eVendas/Dominio/Cartao.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Dominio
{
    public class Cartao
    {
        private int idCartao;

        public int IdCartao
        {
            get { return idCartao; }
            set { idCartao = value; }
        }

        private Cliente cliente;

        public Cliente Cliente
        {
            get { return cliente; }
            set { cliente = value; }
        }

        private decimal limite;

        public decimal Limite
        {
            get { return limite; }
            set { limite = value; }
        }

    }
}
